tests/functional/test_logger.py

```diff
@@ -460,6 +460,32 @@ def handler(event, _):
     assert request_id == log["correlation_id"]
 
 
+def test_logger_set_correlation_id_to_none(lambda_context, stdout, service_name):
+    # GIVEN a logger with a correlation_id set
+    logger = Logger(service=service_name, stream=stdout)
+    logger.set_correlation_id("foo")
+
+    # WHEN calling set_correlation_id with None
+    logger.set_correlation_id(None)
+
+    # THEN there should be no correlation_id
+    logger.info("Foo")
+    log = capture_logging_output(stdout)
+    assert "correlation_id" not in log
```

Comment:
That's not needed ;) https://github.com/awslabs/aws-lambda-powertools-python/blob/89e8151b5c930ba79d685d05a83ae362cf94839d/tests/functional/test_logger_powertools_formatter.py#L175

Reply:
done